control/control_performance_analysis/src/control_performance_analysis_core.cpp

```diff
@@ -38,7 +38,7 @@ ControlPerformanceAnalysisCore::ControlPerformanceAnalysisCore()
   p_.wheelbase_ = 2.74;
```

Comment:
When designing 1d filter, we use xdot = -1/tau (x - u) as the differential equation. The cut-off frequency of this system is 1/tau and in the Laplace domain the filter transfer function is defined by wc/(s+wc). 
The lpf gain parameter might create confusion as it did for me. The gain of filter is always 1 (we want to keep the input output amplitude ratio 1. ).

Reply:
> There is room to improve code optimization, such as using const in references wherever possible.

- I optimized the variables declaration.

> The lpf gain parameter might be confusing, instead I recommend to implement a LPF by cut-off frequency, differential equation and discrete state space update.

- I added this as future improvements because of lack of time. I will open a PR for this as soon as possible.